### PR TITLE
Update docs for Google Analytics 4

### PR DESCRIPTION
**Pull request by nickserv**
### What?

Update docs for Google Analytics 4

### Why?

This page doesn't make it clear which script is recommended, and has code that's outdated and erroring.

### How?

Update headings and clean up bad code.

### CODE REVIEW

File: errors/next-script-for-ga.md [hunk 1]
```diff
@@ -49,7 +45,7 @@ import Script from 'next/script'
 function Home() {
   return (
     <div className="container">
-      <Script id="google-analytics" strategy="afterInteractive">
+      <script>
```
Comment (leerob) [inline errors/next-script-for-ga.md:48]: This should still use `Script`

File: errors/next-script-for-ga.md [hunk 2]
```diff
@@ -18,28 +18,24 @@ import Script from 'next/script'
...
   return (
     <div className="container">
-      <!-- Global site tag (gtag.js) - Google Analytics -->
-      <Script
-        src="https://www.googletagmanager.com/gtag/js?id=GA_MEASUREMENT_ID"
-        strategy="afterInteractive"
-      />
-      <Script id="google-analytics" strategy="afterInteractive">
+      <Script src="https://www.googletagmanager.com/gtag/js?id=GA_MEASUREMENT_ID" />
+      <script>
```
Comment (leerob) [inline errors/next-script-for-ga.md:22]: This should use `Script` as well

**Comment by nickmccurdy** [comment]
@leerob I forgot that `Script` is necessary to prevent hydration errors, so I'll fix this for now. However, I'm concerned that if users try to call the `gtag` or `ga` function, is may not be defined because these scripts are async. Is there a way we can make these scripts synchronous while still supporting hydration, or is this not an issue?

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 022b610f5ff9d1a3999cba56b05729573356af21

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 022b610f5ff9d1a3999cba56b05729573356af21

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Review by ijjk — APPROVED** [review]

**Comment by leerob** [comment]
Can you explain why `afterInteractive` was removed?

**Comment by nickmccurdy** [comment]
Because it's the default behavior, and I wanted to make the code easier to read and understand.

**Pull request merged**